### ISSUE TITLE
Measure CPU required by a container

### ISSUE THREAD

**Issue by gomezgoiri** [PTAnywhere/performance-testing#9]
So far we register the total CPU used (see commit c4510f2ead1ee782bad0712027da175dfd4a4088).

**Comment by gomezgoiri** [comment]
References:
- [CPU usage as percentage in Docker](http://stackoverflow.com/questions/30271942/get-docker-container-cpu-usage-as-percentage)

**Comment by gomezgoiri** [comment]
I'm registering 0% usage by each container when the host machine has the CPU at more than 90% (checked using _top_ command) and the most consuming _Packet Tracer_ at each moment have around 5% of the usage.

Furthermore, Docker is returning timeouts from 200 containers on (issue #10).

Therefore, I'm sure that I am not registering the appropriate percentage.

**Comment by gomezgoiri** [comment]
The bug was in a integer division which returned 0 (as an integer, ignoring decimals). See commit 5e598f1a34a60430335bba30ea77e51060dad2af.

I have tested and returns the same value as `docker stats`.

**Comment by gomezgoiri** [comment]
I have tried to tweak xvfb's configuration parameters and see how it would affect the CPU.

To do this, I have used the same container and change _supervisor.conf_.

Although it seems that it affects it somehow, I could not obtain big improvements.
- Original (1024x768x16): always around 6% (5.5-6.3)
- 1x1x8: Around 30%
- 1x1x16:
  - Around 5.5 or even 5% (sometimes reaching 4.sth%)
  - In other runs it was closer to 5.5-6.sth.
- 1x1x24: 5.5-6%

Conclusions:
- The use of 8 bits of depth demands more CPU.
- Less resolution might result in a super-slight improvement of the CPU consumption. However, this appreciation is unreliable (it is based on manual appreciation of _docker inspect_ and it changes every second) and would require more tests. Furthermore, interacting with the webapp also changes this values.
